backend_interactors/EffectorBackendInteractor.py

```diff
@@ -22,8 +22,10 @@ def run(self):
             print(effector_control_event)
             try:
                 for effector in self._effectors:
+                    print(type(effector))
                     if effector.get_effector_id() == int(effector_control_event["effectorId"]):
-                        effector.activate_effector(effector_control_event)
+                        effector.send_controll_to_effector(effector_control_event)
+                        break
                 
             except:

```

Comment:
Could you determine which exceptions should be handled ( I assume not every exception should be ).

Reply:
Try catch is just for protection so that when any error occurs (within effector, or maybe effector control event dict is incorrect) program doesn't stop, so there are multiple exceptions that can be thrown